### PR TITLE
[zh-cn]: update the translation of Window `back()` method

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Window/back
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/window/back/index.md
* Last commit: https://github.com/mdn/content/commit/44c4ec928281dc2d7c5ea42b7d2c74a2013f16ac

### CODE REVIEW

File: files/zh-cn/web/api/window/back/index.md [hunk 1]
```diff
@@ -1,24 +1,52 @@
...
+
+无。
+
+### 返回值
+
+无（{{jsxref("undefined")}}）。
+
+## 示例
+
+这个简单的示例通过调用 `back()` 来处理“返回”按钮的点击操作。
```
Comment (T34-active) [inline files/zh-cn/web/api/window/back/index.md:31]:
```suggestion
这个简单的示例通过将调用 `back()` 来处理“返回”按钮的点击操作。
```

File: files/zh-cn/web/api/window/back/index.md [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
 ---
 
 {{APIRef}}{{ Non-standard_header() }}{{deprecated_header}}
 
-## 总结
+{{domxref("window")}} 接口上过时的非标准方法 `back()` 会将窗口返回到历史记录中的前一个记录。这是 Firefox 特有的方法，已在 Firefox 31 中删除。
 
-跳转窗口到 history 中的前一个地址，这曾是 Gecko 的方法。请使用标准的 history.back 替代它。
+> [!NOTE]
+> 请使用标准的 {{domxref(“history.back”)}} 方法。
```
Comment (yin1999) [inline files/zh-cn/web/api/window/back/index.md:13]:
```suggestion
> 请使用标准的 {{domxref("history.back")}} 方法。
```

File: files/zh-cn/web/api/window/back/index.md [hunk 3]
```diff
@@ -1,24 +1,52 @@
...
+
+无。
+
+### 返回值
+
+无（{{jsxref("undefined")}}）。
+
+## 示例
+
+这个简单的示例通过将调用 `back()` 来处理“返回”按钮的点击操作。
```
Comment (yin1999) [inline files/zh-cn/web/api/window/back/index.md:31]:
```suggestion
这个简单的示例通过调用 `back()` 来处理“返回”按钮的点击操作。
```

File: files/zh-cn/web/api/window/back/index.md [hunk 4]
```diff
@@ -1,24 +1,52 @@
...
+function handleMyBackButton() {
+  if (canGoBack) {
+    window.back();
+  }
 }
 ```
+
+## 规范
+
+这不属于任何规范的一部分。
```
Comment (yin1999) [inline files/zh-cn/web/api/window/back/index.md:43]:
```suggestion
不属于任何规范。
```

File: files/zh-cn/web/api/window/back/index.md [hunk 5]
```diff
@@ -1,24 +1,52 @@
...
+
+## 浏览器兼容性
+
+这种非标准方法只在 Firefox 中实施，在 Firefox 31 中被删除。
```
Comment (yin1999) [inline files/zh-cn/web/api/window/back/index.md:47]:
```suggestion
这种非标准方法只在 Firefox 中实现过，并在 Firefox 31 中被删除。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]

**Pull request merged**